arm.Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM mcr.microsoft.com/dotnet/core/aspnet:2.2-stretch-slim-arm32v7 AS base
+WORKDIR /app
+EXPOSE 80
+
+COPY /src/BaGet/publish/arm .
+ENTRYPOINT ["dotnet", "BaGet.dll"]
```

Comment:
You can't build this Dockerfile without first installing all the development dependencies (like .NET SDK, NPM, etc..). The default Dockerfile manages all that for you so that you don't have to worry about it. Could you follow the model of the default dockerfile instead?

Reply:
Of course I could follow the default model of docker, but if I did not have the sources I could not compile the dockerfile, just try to make a solution that runs on architecture of x86_64 (DevOps) and generate an image for arm32v7.

I already tried the model, but I did not succeed, and that is how I achieved it.

Reply:
I'm Sorry, this dockerfile is for the job that i added, and not for existed Docker Job